ui/helper/GraphQLWellKnownTypes.js

```diff
@@ -0,0 +1,14 @@
+export const wellKnownTypes = [
```

Comment:
I wonder if it would make more sense to define these in `ui/graphql/types/WellKnownTypes.js` instead.